File: code/accident_a1.c
#include <assert.h>
#include <stdbool.h>

bool accident_detected(double vel, double accel) {
  if (vel >= 15 && accel >= 4) {
    return true;
  }
  return false;
}

int main() {
  assert(accident_detected(15, 4));
  assert(accident_detected(16, 5));
  assert(!accident_detected(13, 4));
  assert(!accident_detected(15, 3));
  return 0;
}

File: code/accident_a1.py
"""Code snippet for scenario a1"""


def accident_detected(vel: float, accel: float) -> bool:
    """Determines whether or not an accident has occurred"""
    if vel >= 15 and accel >= 4:
        return True
    return False


assert accident_detected(15, 4)
assert accident_detected(16, 5)
assert not accident_detected(13, 4)
assert not accident_detected(15, 3)
